src/main/java/WayofTime/alchemicalWizardry/client/nei/NEIMeteorRecipeHandler.java

```diff
@@ -61,10 +65,50 @@ public CachedMeteorRecipe(MeteorParadigm meteor, ItemStack focusStack) {
                     row++;
                 }
 
-                if (focusStack != null && matchItem(focusStack, stack)) {
+                if (matchItem(focusStack, stack)) {
                     this.focus = new Point(xPos - 1, yPos - 1);
                 }
             }
+
+            List<MeteorParadigmComponent> sortedFiller = new ArrayList<>(meteor.fillerList);
```

Comment:
Move this inside of the conditional below. (Or to the top of the method, as I mentioned in an earlier comment.)

Reply:
Addressed in [49e0274](https://github.com/GTNewHorizons/BloodMagic/pull/60/commits/49e0274d9ea18e2c9db6679ef36285538417cc92)